scad/parameters.scad

```diff
@@ -16,9 +16,12 @@ pcb_type = "printed";  // [printed, traditional]
 switch_type = "mx";  // [mx, choc]
 // Switch orientation (based on LED location)
 switch_orientation = "south";  // [north, south]
+// Whether or not to print the side holes (for the extra two pins of a 5-pin switch).
+five_pin_switch = true;
 // Whether to use experimental diode leg contact
 use_folded_contact = false;
-
+// Angle the wire crosses the top switch pin
+top_pin_wire_kink_angle = 15;
```

Comment:
Looking at the output, it feels like the default angle should be negative, since that avoids potential structural issues with the top of the channel. I see that it interferes a bit with the choc diode channel, but wrapping that channel the other way around the central pin should fix that.

Reply:
Avoiding structural issues was why I suggested an upper bound of 35 in the docs, should probably have added that here too.

Negative hits the mx diode pin around 35°, positive hits the cutout edge around 40° so that's why I went with positive.

I have no other preference, but I'd be uncomfortable altering the choc shape. If you want to do that, would you be happy making the change yourself?

Reply:
yeah I can try to change the side that the choc diode wraps

Reply:
Thanks. I don't have any choc switches with which to test this

Reply:
I'll leave the choc diode change to you in a separate PR I think.